src/app/pages/client/space/Space.tsx

```diff
@@ -385,57 +518,93 @@ export function Space() {
-                  >
-                    <div style={{ paddingTop: vItem.index === 0 ? undefined : config.space.S400 }}>
+        </Box>
+        <NavCategory
+          style={{
+            height: `${virtualizer.getTotalSize()}px`,
+            width: '100%',
+            position: 'relative',
+          }}
+        >
+          {virtualizer.getVirtualItems().map((vItem) => {
+            const item = processedHierarchy[vItem.index];
+            if (!item) return null;
+            const renderContent = () => {
+              switch (item.type) {
+                case 'category': {
+                  const { room, categoryId } = item;
+                  const { name } = room;
+                  const paddingTop = config?.space?.S400 ?? '1rem';
+                  return (
+                    <div style={{ paddingTop }}>
                       <NavCategoryHeader>
                         <RoomNavCategoryButton
                           data-category-id={categoryId}
                           onClick={handleCategoryClick}
                           closed={closedCategories.has(categoryId)}
                         >
-                          {roomId === space.roomId ? 'Rooms' : room?.name}
+                          {name}
                         </RoomNavCategoryButton>
                       </NavCategoryHeader>
                     </div>
-                  </VirtualTile>
-                );
+                  );
+                }
+                case 'room_header': {
+                  const { title, categoryId } = item;
+                  return (
+                    <Box>
+                      <NavCategoryHeader variant="Subtle">
+                        <RoomNavCategoryButton
+                          data-category-id={categoryId}
+                          onClick={handleCategoryClick}
+                          closed={closedCategories.has(categoryId)}
+                        >
+                          {title}
+                        </RoomNavCategoryButton>
+                      </NavCategoryHeader>
+                    </Box>
+                  );
+                }
+                case 'room': {
+                  const { room } = item;
+                  return (
+                    <Box>
+                      <RoomNavItem
+                        room={room}
+                        selected={selectedRoomId === room.roomId}
+                        showAvatar={mDirects.has(room.roomId)}
+                        direct={mDirects.has(room.roomId)}
+                        linkPath={getToLink(room.roomId)}
+                        notificationMode={getRoomNotificationMode(
+                          notificationPreferences,
+                          room.roomId
+                        )}
+                      />
+                    </Box>
+                  );
+                }
+                case 'user': {
+                  const { sender, room } = item;
+                  return (
+                    <Box>
```

Comment:
Improve call member display
```suggestion
                    <Box style={{ paddingLeft: config.space.S200 }}>
```